### ISSUE TITLE
No video formats found error for youtube link 'https://www.youtube.com/watch?v=_CMBCfxN1lU'

### ISSUE THREAD

**Issue by arpitpattewar** [ytdl-org/youtube-dl#29654]
## Checklist

- [ ] I'm reporting a broken site support issue
- [ ] I've verified that I'm running youtube-dl version **2021.06.06**
- [ ] I've checked that all provided URLs are alive and playable in a browser
- [ ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ ] I've searched the bugtracker for similar bug reports including closed ones
- [ ] I've read bugs section in FAQ

## Verbose log

```
youtube_dl.utils.DownloadError: ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.```


## Description

<!--
Provide an explanation of your issue in an arbitrary form. Please make sure the description is worded well enough to be understood, see https://github.com/ytdl-org/youtube-dl#is-the-description-of-the-issue-itself-sufficient. Provide any additional information, suggested solution and as much context and examples as possible.
If work on your issue requires account credentials please provide them or explain how one can obtain them.
-->

WRITE DESCRIPTION HERE

**Comment by kikuyan** [comment]
> Be sure to call youtube-dl with the --verbose flag and include its complete output.

Do not ignore this very important part.

**Comment by dirkf** [comment]
Works for me: 
```
youtube-dl -v -F 'https://www.youtube.com/watch?v=_CMBCfxN1lU'[debug] System config: ['--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.youtube.com/watch?v=_CMBCfxN1lU']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.5.2 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[youtube] _CMBCfxN1lU: Downloading webpage
[youtube] _CMBCfxN1lU: Downloading player 408a20d8
[info] Available formats for _CMBCfxN1lU:
format code  extension  resolution note
249          webm       audio only tiny   44k , webm_dash container, opus @ 44k (48000Hz), 1.30MiB
250          webm       audio only tiny   56k , webm_dash container, opus @ 56k (48000Hz), 1.64MiB
251          webm       audio only tiny  102k , webm_dash container, opus @102k (48000Hz), 2.98MiB
140          m4a        audio only tiny  129k , m4a_dash container, mp4a.40.2@129k (44100Hz), 3.76MiB
394          mp4        256x144    144p   65k , mp4_dash container, av01.0.00M.08@  65k, 25fps, video only, 1.91MiB
160          mp4        256x144    144p   67k , mp4_dash container, avc1.4d400c@  67k, 25fps, video only, 1.97MiB
278          webm       256x144    144p   87k , webm_dash container, vp9@  87k, 25fps, video only, 2.53MiB
395          mp4        426x240    240p  104k , mp4_dash container, av01.0.00M.08@ 104k, 25fps, video only, 3.03MiB
133          mp4        426x240    240p  129k , mp4_dash container, avc1.4d4015@ 129k, 25fps, video only, 3.74MiB
242          webm       426x240    240p  143k , webm_dash container, vp9@ 143k, 25fps, video only, 4.17MiB
396          mp4        640x360    360p  198k , mp4_dash container, av01.0.01M.08@ 198k, 25fps, video only, 5.74MiB
243          webm       640x360    360p  259k , webm_dash container, vp9@ 259k, 25fps, video only, 7.52MiB
134          mp4        640x360    360p  326k , mp4_dash container, avc1.4d401e@ 326k, 25fps, video only, 9.46MiB
397          mp4        854x480    480p  337k , mp4_dash container, av01.0.04M.08@ 337k, 25fps, video only, 9.78MiB
244          webm       854x480    480p  438k , webm_dash container, vp9@ 438k, 25fps, video only, 12.70MiB
135          mp4        854x480    480p  649k , mp4_dash container, avc1.4d401e@ 649k, 25fps, video only, 18.83MiB
398          mp4        1280x720   720p  622k , mp4_dash container, av01.0.05M.08@ 622k, 25fps, video only, 18.05MiB
247          webm       1280x720   720p  815k , webm_dash container, vp9@ 815k, 25fps, video only, 23.63MiB
136          mp4        1280x720   720p 1172k , mp4_dash container, avc1.4d401f@1172k, 25fps, video only, 33.99MiB
399          mp4        1920x1080  1080p 1050k , mp4_dash container, av01.0.08M.08@1050k, 25fps, video only, 30.45MiB
248          webm       1920x1080  1080p 1371k , webm_dash container, vp9@1371k, 25fps, video only, 39.77MiB
137          mp4        1920x1080  1080p 2033k , mp4_dash container, avc1.640028@2033k, 25fps, video only, 58.95MiB
18           mp4        640x360    360p  536k , avc1.42001E, 25fps, mp4a.40.2 (44100Hz), 15.55MiB (best)
$
```

**Comment by arpitpattewar** [comment]
Let me try again with --verbose flag.
Thank you.


-Arpit

**Comment by tjadowski** [comment]
For me, this is not working :( and I don't have idea why some video works correctly and some not

```bash
$ youtube-dl -v -F 'https://www.youtube.com/watch?v=_CMBCfxN1lU'
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'https://www.youtube.com/watch?v=_CMBCfxN1lU']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.8.10 (CPython) - Linux-5.8.0-63-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] _CMBCfxN1lU: Downloading webpage
WARNING: Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] _CMBCfxN1lU: Downloading API JSON
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/tom/.local/lib/python3.8/site-packages/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/home/tom/.local/lib/python3.8/site-packages/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/home/tom/.local/lib/python3.8/site-packages/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/home/tom/.local/lib/python3.8/site-packages/youtube_dl/extractor/youtube.py", line 1715, in _real_extract
    self._sort_formats(formats)
  File "/home/tom/.local/lib/python3.8/site-packages/youtube_dl/extractor/common.py", line 1374, in _sort_formats
    raise ExtractorError('No video formats found')
youtube_dl.utils.ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dirkf** [comment]
See #23638.

**Comment by tjadowski** [comment]
You are right, sorry for this, with generated cookies it's works, thanks :pray:
